src/HW05.java

```diff
@@ -0,0 +1,94 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class HW05 {
+    public static void main(String[] args) {
+        HW05 hw05 = new HW05();
+        System.out.println(hw05.sumOfNumbers(new int[]{13, 34, 78, 34}));
+        List<Integer> ints = new ArrayList<>();
+        ints.add(20);
+        ints.add(30);
+        ints.add(100);
+        System.out.println(average(ints));
+        HW05.multiplicationTable(10);
+        fizzBuzz();
+        List<String> firstString = new ArrayList<>();
+        firstString.add("a");
+        firstString.add("b");
+        firstString.add("c");
+        List<String> secondString = new ArrayList<>();
+        secondString.add("1");
+        secondString.add("2");
+        secondString.add("3");
+        System.out.println(getTwoStrings(firstString, secondString));
+
+
+    }
+
+
+    // task 1
+    public int sumOfNumbers(int[] a) {
+        return Arrays.stream(a).sum();
```

Comment:
Did you do it yourself? How did you find about this solution? We haven't learned yet about the streams, hopefully you understand what you did here

Reply:
i write another code, but intellij helps me with  ctrl + alt + L.  and yes, i won`t hide, sometimes i use chatgpt  because I have problems with writing. when I look at the theory I understand everything. but it’s still difficult to write for me.